src/lib/eliom_common.server.ml

```diff
@@ -1338,6 +1341,11 @@ module Persistent_tables = struct
       ) !functorial_tables
     >>= fun functorial_counts ->
     Lwt.return @@ polymorphic_counts @ functorial_counts
+
+  let omit_storage_key = Lwt.new_key ()
+  let do_omitting_storage b = Lwt.with_value omit_storage_key (Some b)
```

Comment:
Currently, request-specific information is stored using key `Eliom_common.sp_key` in a value of type `Eliom_common.server_params`. Could we just extend this type?

Reply:
There is an even better solution. We can simply evaluate the headers from the server parameters.